pytest_watch/watcher.py

```diff
@@ -119,10 +119,18 @@ def _reduce_events(events):
     return filtered
 
 
+def _bright(arg):
+    return STYLE_BRIGHT + arg + Style.RESET_ALL
+
+
+def _highlight(arg):
+    return STYLE_HIGHLIGHT + arg + Style.RESET_ALL
+
+
 def _show_summary(argv, events, verbose=False):
     command = ' '.join(argv)
-    bright = lambda arg: STYLE_BRIGHT + arg + Style.RESET_ALL
-    highlight = lambda arg: STYLE_HIGHLIGHT + arg + Style.RESET_ALL
```

Comment:
___PEP8-check___
/mnt/fastdata/rootserver/git/repo/external/pytest-watch/pytest_watch/watcher.py:125:5: E731 do not assign a lambda expression, use a def
    highlight = lambda arg: STYLE_HIGHLIGHT + arg + Style.RESET_ALL